src/site/artifacts/build.rs

```diff
@@ -0,0 +1,67 @@
+use crate::config::Config;
```

Comment:
i might call this file "page" and then have it have a build command

Reply:
done!